src/lib/server-only-api/crew.ts

```diff
@@ -24,3 +24,15 @@ export async function getCrews(pageIndex: number, pageSize: number) {
         .range(pageIndex * pageSize, (pageIndex + 1) * pageSize - 1)
         .returns<Crew[]>();
 }
+
+export async function getCrewForCrewHomepage({ workId }: { workId: string }) {
+    const supabaseServerClient = await createSupabaseServerClient();
+    return await supabaseServerClient
+        .from('works')
+        .select(
+            'id, title, description, crew:crew_id (id, title, root_work:root_id (id, description))'
```

Comment:
```suggestion
            'id, crew:crew_id (id, title, root_work:root_id (id, description))'
```

why do you need this data?